docs/source/index.rst

```diff
@@ -28,11 +28,17 @@ Features described in this documentation are classified by release status:
   binary distributions like PyPI or Conda, except sometimes behind run-time
   flags, and are at an early stage for feedback and testing.
 
+.. toctree::
+   :glob:
+   :maxdepth: 1
+   :caption: Community
+
+   community/*
 
 .. toctree::
    :glob:
    :maxdepth: 1
-   :caption: Notes
+   :caption: Guides
```

Comment:
Also is it expected that this section is auto-expanded on the landing page? it wasn't before.

Reply:
@albanD I did not change this behavior in this PR. I think it is controlled by this property in the conf.py file: https://github.com/pytorch/pytorch/blob/master/docs/source/conf.py#L412

Reply:
Couldn't it be a side effect of the rename/re-order done in this PR?
I only observed that the 1.11 doc does not do that while the master one does. And I don't think there is any other PR touching this since 1.11?

Reply:
Yes, you're right. I think this should fix it: https://github.com/pytorch/pytorch/pull/78552.